docs/how-to-create-a-new-class-for-source-code.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "How to create a new class for source code"
+source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/how-to-create-a-new-class-for-source-code.md'
+category: Development
+version: v6.0
+tags: [howto, developers, enterprise, community]
+description: This article describes how to create a new class for containing custom source code.
+---
+# Steps
+In order to have your code running in Sense/Net you will have to place the compiled code in the bin folder. There are two ways to achieve this.
+
+## Option 1: Add new class in the web application
+If you have not yet already set up your development environment please read [How to set up development environment](how-to-set-up-development-environment.md). Open the web application that is set up as the IIS site for your Sense/Net installation (_MyWebApplication_) and add a new class to the `Code` folder (see section _Create Code and Root folders_ in [How to set up development environment](how-to-set-up-development-environment.md)). Since your code will be part of the running web application, rebuilding your web application is adequate to have your compiled code running in the bin folder.
```

Comment:
```suggestion
If you have not yet already set up your development environment please read [How to set up development environment](how-to-set-up-development-environment.md). Open the web application that is set up as the IIS site for your sensenet installation (_MyWebApplication_) and add a new class to the `Code` folder (see section _Create Code and Root folders_ in [How to set up development environment](how-to-set-up-development-environment.md)). Since your code will be part of the running web application, rebuilding your web application is adequate to have your compiled code running in the bin folder.
```